controls/tests.py

```diff
@@ -208,13 +204,13 @@ def test_component_download_oscal_json(self):
         elif os.path.isfile(self.json_download.name):
             os.remove(self.json_download.name)
         self.click_element("a#oscal_download_json_link")
-        var_sleep(2)            # need to wait for download, alas
+        var_sleep(1)
```

Comment:
Should all calls to `var_sleep` just be replaced with `wait_for_sleep_after`? If not, I'm just curious why not.

Reply:
I am not sure why but especially file uploads/downloads are fickle and still need the var_sleeps.